yarn.lock

```diff
@@ -13585,6 +13599,19 @@ react-portal@^4.1.2:
   dependencies:
     prop-types "^15.5.8"
 
+react-redux@5:
```

Comment:
mmh, need to checkout the yarn.lock changes, none required.

Reply:
Done